scripts/histmakers/mz_dilepton.py

```diff
@@ -25,13 +25,8 @@
 parser.add_argument("--bias-calibration", action='store_true', help="Adjust central value by calibration bias hist")
 parser.add_argument("--smearing", action='store_true', help="Smear pT such that resolution matches data")
 
-f = next((x for x in parser._actions if x.dest == "pt"), None)
-if f:
-    newPtDefault = [44,26.,70.]
-    logging.warning("")
-    logging.warning(f" >>> Modifying default of {f.dest} from {f.default} to {newPtDefault}")
-    logging.warning("")
-    f.default = newPtDefault
+parser = common.set_parser_default(parser, "pt", [44,26.,70.])
```

Comment:
This can probably go also for the Wlike histogram maker, but in that case the upper range should be 60 (btw general question for everybody: do we have enough Z bosons to fit eta-pt until 60 GeV? )

Reply:
Yes I will add it for the wlike as well and it depends if we have enough MC stat, here is an unrolled plot up to 60:
https://dwalter.web.cern.ch/dwalter/WMassAnalysis/230130_mz/unrolled_nominal_all_muonCorrMassFit_upperPtCut60.png

Reply:
To me it looks fine, can always change it later